english/cry/cry.md

```diff
@@ -0,0 +1,3 @@
+As you cry and cry, your tears fill the cup you're holding and soon you have a cup full of salty tears. And you can't do much with that!
+
+[Why don't you just call your mom?](mom/mom.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.